src/app/workspace/resource/representation/video/video.component.html

```diff
@@ -9,6 +9,24 @@
             (canplaythrough)="loadedVideo()">
         </video>
 
+        <div class="video-overlay" *ngIf="!play && firstPlayed">
```

Comment:
The video overlay should only show when hovering over the video no matter if it's playing or not. The play button should appear if it's paused and vice versa.

Reply:
https://github.com/dasch-swiss/dsp-app/pull/798/commits/4b8f22709eba71aac7ece4cbc16b0760687ef1c4